plugin/api.go

```diff
@@ -42,6 +42,11 @@ type API interface {
 	// CreateUser creates a user.
 	CreateUser(user *model.User) (*model.User, *model.AppError)
 
+	// CreateDirectChannel creates a Direct channel.
```

Comment:
```suggestion
	// CreateDirectChannel creates a direct message channel between two users.
```

Working taken from https://api.mattermost.com/#tag/channels%2Fpaths%2F~1channels~1direct%2Fpost

Reply:
done, thanks